theme/themes/pastanaga/extras/contents.less

```diff
@@ -125,3 +125,15 @@
     background-color: #edf1f2 !important;
   }
 }
+
+#content-core {
```

Comment:
@iFlameing this selector is not specific enough.

Reply:
I pushed the changes.